pkg/sql/tests/monotonic_insert_test.go

```diff
@@ -214,6 +213,10 @@ RETURNING val, sts, node, tb`,
 			t.Errorf("'val' column is not unique: %d results, but %d distinct:\n%s",
 				len(results), numDistinct, results)
 		}
+
+		if len(results) < len(tc.Conns)*2 {
+			t.Errorf("test succeeded less queries then number of concurrent connections")
+		}
```

Comment:
This part should go away because it demonstrates that test will be flaking because it gets into yet unexplained wait for X seconds.